packages/openactive-broker-microservice/app.js

```diff
@@ -884,17 +891,29 @@ async function startPolling() {
 
   const harvesters = [];
 
+  const isParentFeed = {
```

Comment:
Totally optional as it doesn't affect functionality, just readability:

Calling this `isParentFeed` feels slightly misleading as it's being used as a 3-branch conditional below, and then just as a list of feeds in error statement.

Maybe slightly more readable would be:
```
const additionalTypeParentNonParentMap = {
https://openactive.io/SessionSeries': PARENT_ADDITIONAL_FEED,
....
```
and 
```
if (additionalTypeParentNonParentMap[dataDownload.additionalType] === PARENT_ADDITIONAL_FEED)
```
etc

imo `Object.keys(isParentFeed).map((x)` is a bit misleading as we're using it for all feeds, not just parent ones

Reply:
Oh sure - is `PARENT_ADDITIONAL_FEED` a `Symbol` here?

Reply:
(Merging now for speed)

Reply:
>Oh sure - is PARENT_ADDITIONAL_FEED a Symbol here?

Yeh exactly